src/Data/Int.purs

```diff
@@ -77,6 +79,21 @@ foreign import toNumber :: Int -> Number
 fromString :: String -> Maybe Int
 fromString = fromStringAs (Radix 10)
 
+-- | A type for describing whether an integer is even or odd.
+data Parity = Even | Odd
+
+derive instance eqParity :: Eq Parity
+derive instance ordParity :: Ord Parity
```

Comment:
Maybe I should have put this in the Prelude then - I did consider that at first, but I couldn't find much info about parity being used for anything other than integers, I thought EuclideanRing or something maybe... but yeah, I didn't look too hard either ;)